src/components/evidence-tag.jsx

```diff
@@ -31,12 +50,22 @@ EvidenceTag.propTypes = {
   /**
    * Decides the colour of the tag
    */
-  reviewed: PropTypes.bool,
+  className: PropTypes.string,
+  /**
+   * Decides the colour of the tag
+   */
+  IconComponent: PropTypes.element,
```

Comment:
Getting a PropTypes error when a IconComponent is provided (`IconComponent: ConfigureIcon`). Can either:
1. change PropType to `PropTypes.func`.
2. keep PropType as an element but use `{cloneElement(IconComponent, { width: size, height: size })}` where it's used at L22. Then an element must be passed eg (`IconComponent: < ConfigureIcon />`). Note that the default would need to become `<Tag />`.

Reply:
Very nice! Went for 2. and updated the prop name to be `kebabCase`